modules/pricer/src/main/java/com/opengamma/strata/pricer/bond/DiscountingBillTradePricer.java

```diff
@@ -236,6 +247,20 @@ public CurrencyAmount currentCash(ResolvedBillTrade trade, LocalDate valuationDa
     return CurrencyAmount.zero(trade.getProduct().getCurrency());
   }
 
+  //-------------------------------------------------------------------------
+  /**
+   * Calculates the settlement date.
```

Comment:
Add that in absence of settlement date (position?), the valuation date is returned.

Reply:
Javadoc updated